tasks/system_package_management.yml

```diff
@@ -0,0 +1,70 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+- name: Get ownCloud APT OpenPGP key
+  apt_key:
+    url: '{{ owncloud__apt_repo_key_url }}'
```

Comment:
I included the pubkey for now until this is fixed upstream. Thanks.

Tracked upstream: [ownCloud](https://github.com/owncloud/core/issues/25710), [NextCloud](https://github.com/nextcloud/server/issues/755)